string type
---
string type
roscpp_message_reflection/MessageInfo[] infos
